.vscode/settings.json

```diff
@@ -8,10 +8,11 @@
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true,
   "files.trimTrailingWhitespace": true,
+  // https://github.com/microsoft/vscode-black-formatter/issues/82#issuecomment-1155492426
```

Comment:
[FYI that this file supports comments.](https://code.visualstudio.com/Docs/languages/json#_json-with-comments)